tools/db_bench_tool.cc

```diff
@@ -884,6 +884,25 @@ DEFINE_bool(rate_limiter_auto_tuned, false,
             "Enable dynamic adjustment of rate limit according to demand for "
             "background I/O");
 
+
+DEFINE_bool(sine_write_rate, false,
+            "Use a sine wave write_rate_limit");
+
+DEFINE_int32(sine_write_rate_interval_milliseconds, 10000,
+              "Interval of which the sine wave write_rate_limit is recalculated");
```

Comment:
What's the benefit of using int32, rather than uint64?

Reply:
I initially did uint32, but that broke a build. However, since other flags like `rate_limit_delay_max_milliseconds` was set to int32 I figured it would make sense to use the same type for another milliseconds flag. But I'll change it to uint64, then we will avoid unnecessary casting.